src/excmds.ts

```diff
@@ -462,6 +462,7 @@ let LAST_USED_INPUT: HTMLElement = null
  *
  * @param nth   focus the nth input on the page, or "special" inputs:
  *                  "-l": last focussed input
+ *                  "-n": next to last focused input
```

Comment:
> next to last
A bit ambiguous between "second to last" and "(the) input (occurring/found) after last focused input".

Reply:
How about "input after last focused one"?

Or would you please suggest a clear explanation of `focusinput -n`? Not a native English speaker here.